app/views/course/achievement/course_users/_course_users_form.html.slim

```diff
@@ -2,37 +2,17 @@
   = f.error_notification
   = f.error :achievement
 
-  - collection = course.course_users.without_phantom_users.students.order_alphabetically
-  table.table.table-hover
-    thead
-      th = t('.name')
-      th
-        input.toggle-all id="toggle-all-normal-achievements" type='checkbox' style='display: none'
-        '
-        label for='toggle-all-achievements'
-          = t('.enabled')
-    tbody
-      = f.collection_check_boxes :course_user_ids, collection, :id, :name do |f|
-        tr
-          th = link_to_course_user(f.object)
-          td = f.check_box 'data-for-select-all' => "toggle-all-normal-achievements"
+  - own_students = current_course_user.present? ? @course_users.merge(current_course_user.my_students).without_phantom_users : []
```

Comment:
Since there is some loading and scopes, can I suggest we shift this to the controller. Rather than merge in this manner, we can chain the scope: 

```ruby
own_students = current_course_user.present? ? current_course_user.my_students.without_phantom_users.order_alphabetically : []
```

Perhaps we can shorten the above by adding the `.order_alphabetically` automatically into `course_user#my_students` since I cannot think of a case where we need to load `my_students` in any other manner.  Then it becomes:

```ruby
own_students = current_course_user.present? ? current_course_user.my_students.without_phantom_users : []
```